lib/utils/environment.ts

```diff
@@ -0,0 +1,7 @@
+const NODE_ENV = process.env['NODE_ENV'];
+
+export const isProduction = NODE_ENV === 'production';
+
+export const isDevelopment = NODE_ENV === 'development';
+
+export const isTest = Boolean(NODE_ENV === 'test');
```

Comment:
Why `Boolean` here?

Reply:
A hangover from copying much of the functionality from console, simplifying it, but then not simplifying it enough: https://github.com/zendesk/zendesk_console/blob/master/lotus_react/src/utils/environment.ts#L9

Console uses `Boolean` because they use Cypress; at present, zaf_sdk does not.

I can fix this, it is easily enough done.